File: program/check-vowel/CheckVowel.cs
using System;
 
public class vowelChar
{
    static void Main(string[] args)
    {
        char ch;
 
        Console.WriteLine("Enter any character: ");
        ch = Convert.ToChar(Console.ReadLine());
 
        if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u' || ch == 'A' || ch == 'E' || ch == 'I' || ch == 'O' || ch == 'U')
        {
            Console.WriteLine(ch + " is a Vowel.");
        }
        else 
        {
            Console.WriteLine(ch + " is not a Vowel.");            
        }
    }
}

File: program/program/check-vowel/check_vowel.php
<?php
	$str = readline('Enter letter: ');

  if ($str == 'a' 
  || $str == 'e'
  || $str == 'i' 
  || $str == 'o'
  || $str == 'u'
  || $str == 'A'
  || $str == 'E'
  || $str == 'I'
  || $str == 'O'
  || $str == 'U') {
    echo 'Vowel';
  }
  else {
    echo 'Is not a Vowel';
  }
?>
